deployment/configenv/xml_jlibpt/SWEspProcess.cpp

```diff
@@ -26,16 +26,8 @@ SWEspProcess::SWEspProcess(const char* name, EnvHelper * envHelper):SWProcess(na
 {
 }
 
-unsigned SWEspProcess::add(IPropertyTree *params)
+void SWEspProcess::addOtherSelector(IPropertyTree *compTree, IPropertyTree *params)
```

Comment:
is a failure in adding the binding something to worry about?

Reply:
Throw exception